Week1/homework/app.js

```diff
@@ -2,10 +2,104 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'dont_tell_me_your_afraid', 'dont_tell_mummy', 'positive_thinking', 'time_management', 'The alchemist', 'the_power_of_now', 'ekadoli', 'Before_we_were_yours', 'hebrew_female_in_my_heart', 'amareeta'
   ];
 
-
   // Replace with your own code
-  console.log(bookTitles);
+  //  2.3
+  function makeBooksTitlesList() {
+    const ul = document.createElement('ul');
+    ul.innerHTML = 'Books Titles: ';
+    document.getElementsByTagName('body')[0].appendChild(ul);
+    for (let i = 0; i < bookTitles.length; ++i) {
+      const li = document.createElement('li');
+      li.innerHTML = bookTitles[i];
+      ul.appendChild(li);
+    }
+
+  }
+
+  makeBooksTitlesList();
+  //2.4
+
+  const booksDetails = {
+    dont_tell_me_your_afraid: {
+      title: 'don\'t tell me your afraid',
+      language: 'english',
+      author: 'Giuseppe Catozzella',
+    },
+    dont_tell_mummy: {
+      title: 'don\'t tell mummy',
+      language: 'english',
+      author: 'Toni Maguire'
+    },
+    positive_thinking: {
+      title: 'positive thinking',
+      language: 'english',
+      author: 'Lucy Legend'
+    },
+    time_management: {
+      title: 'time management',
+      language: 'arabic',
+      author: 'Majed Khamis'
+    },
+    alchemist: {
+      title: 'The alchemist',
+      language: 'english',
+      author: 'Paulo Coelho'
+    },
+    the_power_of_now: {
+      title: 'The power of now',
+      language: 'german',
+      author: 'Eckhart Tolle'
+    },
+    ekadoli: {
+      title: 'ekadoli',
+      language: 'arabic',
+      author: 'Hanan Lashin'
+    },
+    Before_we_were_yours: {
+      title: 'Before we were yours',
+      language: 'english',
+      author: 'Lisa Wingate'
+    },
+    hebrew_female_in_my_heart: {
+      title: 'hebrew female in my heart',
+      language: 'arabic',
+      author: 'Khawla Hamdi'
+    },
+    amareeta: {
+      title: 'amareeta',
+      language: 'arabic',
+      author: 'Hanan Lashin'
+    }
+
+  };
+  //2.5
+
+  function makeBooksDetailsPage() {
+    const container = document.createElement('div');
+    document.getElementsByTagName('body')[0].appendChild(container);
+    container.innerText = 'Books Details:';
+    for (const obj in booksDetails) {
```

Comment:
These book details should be added to the `li`-s that you created in the first loop. If used `setAttribute()` to set the `id` of each `li` to a book title your `for` loop could iterate through these book titles, get a reference to the corresponding `li` by using `getElementById()` and then append the book details to that `li`.